packages/lean-symbols/1.0.0/_manifest.yml

```diff
@@ -0,0 +1,10 @@
+name: "lean-symbols"
+title: "Unicode Math Lean Symbols"
+description: >
+  Insert symbols like ∑ Π α → ↦  Γ etc, the same set of symbols as the VScode lean4 
+  extension allows you to insert, using the same shortcuts. For example '\a\', '\a ', '\aTAB', 
+  all insert α, i.e. the greek alpha. TAB denotes a single press of the tab key.
+
+version: 0.1.0
```

Comment:
Fails quality check - the path indicates that the version is 1.0.0, but the manifest indicates version 0.1.0.

Reply:
I fixed that now.